core/shared/src/main/scala/zio/stm/TMap.scala

```diff
@@ -630,6 +630,29 @@ object TMap {
    */
   def make[K, V](data: (K, V)*): USTM[TMap[K, V]] = fromIterable(data)
 
+  object unsafe {
+    def make[K, V](data: (K, V)*)(implicit unsafe: Unsafe): TMap[K, V] = {
```

Comment:
Should we modify `allocate` to use this?

Reply:
Yes.